tmva/pymva/test/generateKerasModels.py

```diff
@@ -141,7 +141,20 @@ def generateActivationModel():
     model.compile(loss='mean_squared_error', optimizer=SGD(learning_rate=0.01))
     model.fit(x_train, y_train, epochs=10, batch_size=1)
     model.save('KerasModelActivations.h5')
-    
+
+def generateSwishModel():
+# Create the Keras model
+    model = Sequential()
+    model.add(Dense(64, activation='swish', input_shape=(8,)))
```

Comment:
We might have to check for the version when `swish` was introduced in Keras, and restrict the tests accordingly.